houston/sandbox.py

```diff
@@ -92,8 +92,6 @@ def run(self, mission: Mission) -> MissionOutcome:
                                          action,
                                          state_before,
                                          env)
-                signal.signal(signal.SIGALRM, lambda signum, frame: TimeoutError.produce())
```

Comment:
I'm not sure why we originally decided to use `signal`, but your code seems perfectly fine!